stdlib/public/core/StaticString.swift

```diff
@@ -131,7 +131,7 @@ public struct StaticString
 
   /// Returns a `String` representing the same sequence of Unicode
   /// scalar values as `self` does.
-  @swift3_migration(message="use the 'String(_:)' initializer")
```

Comment:
Huh, I'm not sure why this attribute was removed at all, since stringValue was really removed.  So `message="Use the String(describing:) initializer"` would probably be appropriate here.